ClientApp/viking-vault/src/UserDashboard/TransactionList.tsx

```diff
@@ -30,8 +41,22 @@ class TransactionList extends React.Component<any, IState> {
         transactions: []
     }
 
+    private changeNullUsersToDeleted(result:ITransaction[]): ITransaction[] {
```

Comment:
changeNullUsersToBeDeleted maybe :)

Reply:
Null users are not deleted, just replaced by a mocked user